src/java/org/apache/cassandra/tools/nodetool/Flush.java

```diff
@@ -17,24 +17,35 @@
  */
 package org.apache.cassandra.tools.nodetool;
 
-import io.airlift.airline.Arguments;
-import io.airlift.airline.Command;
-
 import java.util.ArrayList;
 import java.util.List;
 
 import org.apache.cassandra.tools.NodeProbe;
-import org.apache.cassandra.tools.NodeTool.NodeToolCmd;
+import org.apache.cassandra.tools.nodetool.layout.CassandraUsage;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Parameters;
+
+import static org.apache.cassandra.tools.nodetool.CommandUtils.parseOptionalKeyspace;
+import static org.apache.cassandra.tools.nodetool.CommandUtils.parseOptionalTables;
+import static org.apache.cassandra.tools.nodetool.CommandUtils.concatArgs;
 
 @Command(name = "flush", description = "Flush one or more tables")
-public class Flush extends NodeToolCmd
+public class Flush extends AbstractCommand
 {
-    @Arguments(usage = "[<keyspace> <tables>...]", description = "The keyspace followed by one or many tables")
+    @CassandraUsage(usage = "[<keyspace> <tables>...]", description = "The keyspace followed by one or many tables")
     private List<String> args = new ArrayList<>();
 
+    @Parameters(index = "0", description = "The keyspace followed by one or many tables to flush", arity = "0..1")
+    private String keyspace;
+
+    @Parameters(index = "1..*", description = "The tables to flush", arity = "0..1")
```

Comment:
why do we use arity = "0..1" if multiple tables are supported?

Reply:
Fixed.